conjure-java-core/src/main/java/com/palantir/conjure/java/services/JerseyServiceGenerator.java

```diff
@@ -169,9 +170,13 @@ private MethodSpec generateServiceMethod(
         }
 
         if (returnType.equals(BINARY_RETURN_TYPE_OUTPUT) || returnType.equals(BINARY_RETURN_TYPE_RESPONSE)) {
-            methodBuilder.addAnnotation(AnnotationSpec.builder(ClassName.get("javax.ws.rs", "Produces"))
-                    .addMember("value", "$T.APPLICATION_OCTET_STREAM", ClassName.get("javax.ws.rs.core", "MediaType"))
-                    .build());
+            methodBuilder
+                    .addAnnotation(Nullable.class)
```

Comment:
Can we avoid adding `Nullable` to non-optional returns?

Reply:
They are nullable technically tho. But being an annotation I guess it's fine.

Reply:
Returning null for those endpoints violates the conjure defined API